### PR TITLE
Make sure to update the Ubuntu package list before install

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

File: .github/workflows/validate-data.yml [hunk 1]
```diff
@@ -12,7 +12,9 @@ jobs:
         with:
           python-version: 3.6
       - name: Install packages
-        run: sudo apt-get install libxml2-utils apache2-dev libldap2-dev libsasl2-dev libyaml-dev
+        run: |
+          sudo apt-get update
```
Comment (matyasselmeci) [inline .github/workflows/validate-data.yml:16]:
```suggestion
          sudo apt-get update && \
```
or is this already run with set -e?

**Review by matyasselmeci — APPROVED** [review]
LGTM; one comment, feel free to implement or not.

File: .github/workflows/validate-data.yml (hunk 1 above)
Comment (brianhlin) [reply .github/workflows/validate-data.yml:16]: Yup, all GHA runs are! https://github.com/opensciencegrid/topology/runs/1385808985?check_suite_focus=true#step:4:3

**Comment by brianhlin** [comment]
Travis failed due to a transient (?) issue with contacting LIGO's LDAP. This doesn't seem to be the case for GHA so I wonder if LIGO is throttling Travis. Overriding CI and merging.

**Pull request merged**